src/pages/OrderConfirmed.js

```
@@ -0,0 +1,196 @@
+                Votre commande ({items?.length} produits)
+              </Typography>
+              <Box>
+                {items?.map((item) => (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      justifyContent: "space-between",
+                      alignItems: "flex-start",
+                    }}
+                  >
+                    <Box sx={{ mr: 1.5, pl: 2, py: 1.5 }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)